azurerm/resource_arm_azuread_group.go

```diff
@@ -0,0 +1,92 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/Azure/azure-sdk-for-go/services/graphrbac/1.6/graphrbac"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmActiveDirectoryGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmActiveDirectoryGroupCreate,
+		Read:   resourceArmActiveDirectoryGroupRead,
+		Delete: resourceArmActiveDirectoryGroupDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"object_id": {
```

Comment:
Is there a reason for this property to exist? `.id` can be used instead and is even what your using in your tests ` object_id = "${azurerm_azuread_group.test.id}"`